app/presenters/tree_builder_ops_rbac_features.rb

```diff
@@ -0,0 +1,160 @@
+class TreeBuilderOpsRbacFeatures < TreeBuilder
+  has_kids_for Menu::Section,     [:x_get_tree_section_kids]
+  has_kids_for Menu::Item,        [:x_get_tree_item_kids]
+  has_kids_for MiqProductFeature, [:x_get_tree_feature_kids]
+
+  attr_reader :role, :features, :editable, :root_counter
+
+  def initialize(name, type, sandbox, build, role:, editable: false)
+    @role     = role
+    @editable = editable
+    @features = @role.miq_product_features.order(:identifier).pluck(:identifier)
+
+    @root_counter = []
+
+    # Make sure tree_state doesn't hold on to old data between requests
+    TreeState.new(sandbox).remove_tree(name)
+
+    super(name, type, sandbox, build)
+  end
+
+  private
+
+  def filtered_menus
+    menus = Menu::Manager.map do |section|
+      next if section.id == :cons && !Settings.product.consumption
+      next if section.name.nil?
+      next unless Vmdb::PermissionStores.instance.can?(section.id)
+
+      section
+    end
+
+    menus
```

Comment:
The `menus` variable should be omitted and the whole method moved to the [one single place](https://github.com/ManageIQ/manageiq-ui-classic/pull/137/files#diff-590dcdaf22099492b52ecc45ca1b4a7dR51) where it is used.